app/messaging/schemas/schedule.js

```diff
@@ -19,5 +19,6 @@ module.exports = Joi.object({
   scheme,
   documentReference,
   adjustment,
+  remainingAmount: Joi.number().integer().required(),
```

Comment:
Lets create a separate schema for this with specific error messages for each error key. Will also need unit tests for this. E.g. already exist in this repo